### ISSUE TITLE
Colors odd behaviour

### ISSUE THREAD

**Issue by dhoyos-yseop** [yseop/vscode-yseopml#98]
Hi, I'm very happy with the yml extension, but I don't understand the colors, for instance, in a switch case, sometimes it colours pink/white and sometimes green/orange, and sometimes it even mix the colors.

![image](https://user-images.githubusercontent.com/44225796/59111230-3ac04900-8906-11e9-943d-5742acf71f14.png)
![image](https://user-images.githubusercontent.com/44225796/59111264-4ca1ec00-8906-11e9-9265-ac6a2b7e92c2.png)
![image](https://user-images.githubusercontent.com/44225796/59111321-67746080-8906-11e9-9c31-e2e61b78b116.png)

This weird changes of color happens in a IF-ELSE, the element inside the IF are colored pink/white, but the elements at Else are colored green/orange.

![image](https://user-images.githubusercontent.com/44225796/59111077-f339bd00-8905-11e9-9f92-2826bd1ac8e6.png)

**Comment by alacambre-yseop** [comment]
Yeah, we have very odd behaviours with colours. We'll check that soon and hopefully deliver an update on this part.
